dev-lua/mpack/mpack-1.0.3.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Lua bindings for libmpack"
+HOMEPAGE="https://github.com/tarruda/libmpack/"
+SRC_URI="https://github.com/tarruda/libmpack/archive/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/libmpack-${PV}/binding/lua"
```

Comment:
By convention, ``S`` normally appears later in the ebuild:

```
DEPEND="app-foo/bar"
RDEPEND="${DEPEND}"

S="${WORKDIR}/baz"

src_compile() {
```

That said, there there is no formal policy about the placement of ``S`` (and this is no regression compared to 1.0.2) so this comment is purely informational.